jobs/run-inner-mdbci-parametrized-test.yaml

```diff
@@ -0,0 +1,112 @@
+- job:
+    name: run-inner-mdbci-parametrized-tests
+    description: 'This job performs running parametrized tests.'
+    key_one: &SORTED_GIT_BRANCHES_FILE_PATH '/home/vagranttest/mdbci_jenkins_config/branch_list'
```

Comment:
Please replace here and below /home/vagranttest to $HOME - it will increase portability.